android-components/components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounterMenu.kt

```diff
@@ -66,7 +66,7 @@ open class TabCounterMenu(
             text = context.getString(R.string.mozac_browser_menu_new_private_tab),
             start = DrawableMenuIcon(
                 context,
-                iconsR.drawable.mozac_ic_private_mode_24,
+                iconsR.drawable.mozac_ic_private_mode_circle_fill_20,
```

Comment:
I thought the intention was to update the path of the mask_24 icon. Are we sure this was suppose to be used here?

Reply:
swapped it back and updated mask_24 icon path